packages/cli-platform-ios/src/commands/runIOS/index.ts

```diff
@@ -117,6 +118,14 @@ async function runIOS(_: Array<string>, ctx: Config, args: FlagsT) {
     } "${chalk.bold(xcodeProject.name)}"`,
   );
 
+  await runPackager(
```

Comment:
We should be able to disable this with `--no-packager`.

Unrelated: I was actually hoping we could remove the packager from the `run-android` command instead :P

Reply:
> We should be able to disable this with --no-packager.

We're already able to this :) See here:
https://github.com/react-native-community/cli/blob/ba362e65e2a7feebf49a2b392f9900ca3260efdf/packages/cli-platform-ios/src/commands/runIOS/index.ts#L609-L612

And in function body I check for this:
https://github.com/react-native-community/cli/blob/b3741451e5b5c757aeaccc2fb66edaa46faa3d72/packages/cli-plugin-metro/src/commands/start/runPackager.ts#L11-L13

Reply:
Question is, maybe we should consider disabling running packager by default? I think this would require a poll on how people use it in the wild. I myself run the server myself in a terminal tab that suits me. But I've also seen devs relying on automatic opening of dev server and not sure if they would even realize it's missing

Reply:
Other options that comes to my mind is to run the packager in the same window where we start `run-ios/android` command.

Reply:
> I think this would require a poll on how people use it in the wild

FYI I'm up for this, and/or making the most informed decision. Originally I was with @tido64 in thinking we'd drop the behaviour on Android to align. But @szymonrybczak's changes, with an opt-out flag, look pretty useful to me.

While this item of feedback is still unresolved, I'd like to push for this PR to be merged and released anyway, since it is a direct dependency for https://github.com/facebook/react-native/pull/38944 and incoming Debugging functionality in React Native. These are all alpha releases, and we can follow up on enhancing this behaviour in a follow-up PR.|

cc @thymikee